.github/workflows/coverage.yml

```
@@ -0,0 +1,84 @@
+name: NEURON Code Coverage
+
+on:
+  push:
+    branches:
+      - master
+      - live-debug*
+      - release/**
+  pull_request:
+    branches:
+      - master
+      - release/**
+
+jobs:
+  coverage:
+    runs-on: ubuntu-18.04
+
+    name: Code Coverage
+
+    timeout-minutes: 45
+
+    steps:
+
+      - name: Install apt packages
+        run: |
+          sudo apt-get install  build-essential doxygen autoconf automake libtool lcov autotools-dev libopenmpi-dev libmpich-dev libx11-dev libxcomposite-dev mpich openmpi-bin patchelf python-pip python-tk python-numpy python3-numpy python3-pip g++-5 g++-6 g++-8 g++-9
+        shell: bash
+
+      - name: Set up Python2
+        uses: actions/setup-python@v2
+        with:
+          python-version: '2.x'
+
+      - name: Set up Python3
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+
+      - name: Install Python3 dependencies
+        run: python3 -m pip install --upgrade pip wheel setuptools scikit-build pytest matplotlib bokeh ipython cython pytest pytest-cov mpi4py
+
+      - uses: actions/checkout@v2
```

```suggestion
      - uses: actions/checkout@v2
         with:
           fetch-depth: 2
```
forgot to mention that one - needed for codecov reports.